src/promptflow/tests/test_configs/eager_flows/basic_callable_class_with_sample_file/inputs.json

```diff
@@ -0,0 +1 @@
+{"func_input": "mock_input"}
```

Comment:
add some negative example samples
func input is a number or json?
or input is not named as func_input